corehq/apps/receiverwrapper/views.py

```diff
@@ -212,6 +209,10 @@ def secure_post(request, domain, app_id=None):
         'noauth': _noauth_post,
     }
 
+    if should_ignore_submission(request):
+        # silently ignore submission if it meets ignore-criteria
+        return SubmissionPost.get_success_response(None, None)
```

Comment:
Perhaps one option would be to change the response message to be something like: ` √ (this submission was ignored)` - would confirm with Sheel.